Naoshi

anpan3033 opened the pull request:
角煮

anpan3033 committed:
マップにマーカーを表示

anpan3033 committed:
同様

morimoto-aki commented on index.html:
```diff
@@ -1,4 +1,36 @@
+<body>
+  <div id="mapid" style="width: 100%; height: 600px"></div>
+  <script src="https://unpkg.com/leaflet@1.4.0/dist/leaflet.js"></script>
+  <script>
```
ここから34行目までread.jsに移動

anpan3033 committed:
index.htmlはとりあえずこんな感じ

anpan3033 committed:
ｊｓ内でｊｓを読み込むコードに関しては、外部ｊｓを呼び出すコードとかは出てきたんだけれど、それ以外に関してはよくわからなかった。でもｊｓファイルに移す前は表示されていたということは動くことで多分間違いないんだよね？
id="mapid" style="width: 100%; height: 600px"　のmapidが多分ｊｓと同期していないというか引き渡されていないというかそういうの関係ないかな？
ごめん説明下手くそで。
